client/main.go

```diff
@@ -216,10 +227,21 @@ func parseStreamingRatio(streamingRatio string) (int64, int64) {
 }
 
 func sendStreamingRequests(client pb.ResponderClient,
-	shutdownChannel <-chan struct{}, lengthDistribution distribution.Distribution,
+	shutdownChannel <-chan struct{}, clientTimeout time.Duration,
+	lengthDistribution distribution.Distribution,
 	latencyDistribution distribution.Distribution, streamingRatio string,
 	r *rand.Rand, received chan *MeasuredResponse) error {
-	stream, err := client.StreamingGet(context.Background())
+
+	var ctx context.Context
+	if clientTimeout != time.Duration(0) {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithTimeout(context.Background(), clientTimeout)
+		defer cancel()
+	} else {
+		ctx = context.Background()
+	}
+
+	stream, err := client.StreamingGet(ctx)
```

Comment:
Seems weird that there's no effective way to put a timeout on an individual request in a stream. I've removed it and changed the flag doc string.